### ISSUE TITLE
Runtime Commands

### ISSUE THREAD

**Issue by Mentors4EDU** [bernaferrari/FigmaToCode#79]
This runs with Turborepo.
Is there a recommended version?
Can you run this w/ NPM as well?

Docs were a bit vague on runtime/build requirements

**Comment by bernaferrari** [comment]
Yeah, they changed many times and I haven't had time to update the docs, sorry. :(

Turbo-repo is "less needed" (it is just a npm run all, sort of).

Yes, theoretically you can run with NPM, but you probably need to set-up workspaces first. Right now, the recommended way is pnpm.

Just `pnpm i`,  and `pnpm dev` (which calls `turbo dev`). This will build a nextjs website with a debug UI and the plugin, if you want the plugin only, you can just cd to the plugin dir and pnpm dev from there.

After you install from pnpm, you can execute from npm, but I think you need to install the first time using pnpm.

**Comment by xiaoshude** [comment]
how to run the test in local dev

**Comment by bernaferrari** [comment]
The tests were disabled on the last version in favor of the debug UI, so pnpm i and pnpm dev gives everything you need
